cypress/e2e/tests/migration/issues/single-application.test.ts

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    getRandomApplicationData,
+    getRandomAnalysisData,
+    deleteByList,
+} from "../../../../utils/utils";
+import { Analysis } from "../../../models/migration/applicationinventory/analysis";
+import { SEC } from "../../../types/constants";
+let applicationsList: Array<Analysis> = [];
+let application: Analysis;
+
+describe(["@tier2"], "Single application issues inspection", () => {
```

Comment:
This should be tier1

Reply:
Actually same test is covered in https://github.com/konveyor/tackle-ui-tests/blob/main/cypress/e2e/tests/migration/applicationinventory/analysis/source_analysis_without_creds.test.ts#L44. 

Please add validateIssues in this test itself.

Reply:
Moved, resolving this